spec/presenters/chart_presenter_spec.rb

```diff
@@ -31,7 +31,7 @@
   def unique_pageviews_chart_data
     {
       caption: "Unique pageviews from 2018-01-13 to 2018-01-15",
-      chart_id: "unique_pageviews_2018-01-13-2018-01-15_chart",
+      chart_id: "unique_pageviews_chart",
```

Comment:
Could you explain why we have this change in this commit. Seems out of place.

Reply:
Yeah, I made a note of it above: 

`Change the id's of charts and tables to make them easier to query. The date is not an important part of the chart because there will only be one per query and if we display two different time series on the same chart it could be difficult to handle`

Reply:
Sorry, I forgot about your PR description. Do you think It would have been possible to isolate this change in a commit and add those notes in the commit?

Reply:
Sure, done!